src/main/java/frc/robot/maps/Shrimp.java

```diff
@@ -102,6 +107,24 @@ SDSSwerveModule.MK4_V2.wheelDiameter, new PIDValues(0.004, 0.00,
                 config, holonomicDrive);
     }
 
+    @Override
+    public VisionMap getVisionMap() {
+        // Cam mounted 9.029 sideways of center, 9.029 in. forward of center, 9.75 in.
+        // up from center.
+        Transform3d robotToCamFL = new Transform3d(
+                new Translation3d(Units.inchesToMeters(-9.029), Units.inchesToMeters(9.029),
+                        Units.inchesToMeters(9.75)),
+                new Rotation3d(0, Units.degreesToRadians(-45), Units.degreesToRadians(65.752)));
+
+        // Cam mounted facing __, __ forward of center, __ up from center.
+        Transform3d robotToCamFR = new Transform3d(
+                new Translation3d(Units.inchesToMeters(9.029), Units.inchesToMeters(9.029),
+                        Units.inchesToMeters(9.75)),
+                new Rotation3d(0, Units.degreesToRadians(45), Units.degreesToRadians(65.752)));
+        return new VisionMap(new CameraSource("ShrimpCamFL", robotToCamFL),
```

Comment:
A nit-pick but a new line between the `Transform3d` and this line would aid in readability.